src/Akka.Streams.Kafka/Helpers/CommitRefreshing.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Linq;
+using Akka.Streams.Kafka.Stages.Consumers.Actors;
+using Confluent.Kafka;
+
+namespace Akka.Streams.Kafka.Helpers
+{
+    /// <summary>
+    /// This is used by <see cref="KafkaConsumerActor{K,V}"/> to store consumer offsets
+    /// </summary>
+    internal interface ICommitRefreshing<K, V>
+    {
+        void Add(IImmutableSet<TopicPartitionOffset> offsets);
+        void Committed(IImmutableSet<TopicPartitionOffset> offsets);
+        void Revoke(IImmutableSet<TopicPartition> revokedTopicPartitions);
+        IImmutableSet<TopicPartitionOffset> RefreshOffsets { get; }
+        void UpdateRefreshDeadlines(IImmutableSet<TopicPartition> topicPartitions);
+        void AssignedPositions(IImmutableSet<TopicPartition> assignedPartitions, IImmutableSet<TopicPartitionOffset> assignedOffsets);
+        void AssignedPositions(IImmutableSet<TopicPartition> assignedTopicPartitions, IConsumer<K, V> consumer, TimeSpan positionTimeout);
+    }
+
+    internal static class CommitRefreshing
```

Comment:
Seems like this class is used for "commit refresh" feature: storing committed offsets to re-commit them if no new commits were made for some time (set by `commit-refresh-interval` setting and can be disabled).